sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysisResult.java

```diff
@@ -113,64 +112,34 @@ public String getLogs() {
     }
 
     /**
-     * Get a collection of all the sensitivity values in state N.
+     * Get a collection of all the sensitivity values.
      *
-     * @return a collection of all the sensitivity values in state N.
+     * @return a collection of all the sensitivity values.
      */
-    public Collection<SensitivityValue> getSensitivityValues() {
-        return Collections.unmodifiableCollection(sensitivityValues);
+    public Collection<SensitivityValue> getValues() {
+        return Collections.unmodifiableCollection(values);
     }
 
     /**
-     * Get a collection of all the sensitivity values associated with given function in state N.
+     * Get a collection of sensitivity value associated with given contingency id
      *
-     * @param function sensitivity function
-     * @return a collection of all the sensitivity values associated with given function in state N.
-     */
-    public Collection<SensitivityValue> getSensitivityValuesByFunction(SensitivityFunction function) {
-        return sensitivityValues.stream().filter(sensitivityValue -> sensitivityValue.getFactor().getFunction().equals(function))
-                .collect(Collectors.toList());
-    }
-
-    /**
-     * Get a collection of all the sensitivity values associated with given variable in state N.
-     *
-     * @param variable sensitivity variable
-     * @return a collection of all the sensitivity values associated with given variable in state N.
-     */
-    public Collection<SensitivityValue> getSensitivityValuesByVariable(SensitivityVariable variable) {
-        return sensitivityValues.stream().filter(sensitivityValue -> sensitivityValue.getFactor().getVariable().equals(variable))
-                .collect(Collectors.toList());
-    }
-
-    /**
-     * Get the sensitivity value associated with given function and given variable in state N.
-     *
-     * @param function sensitivity function
-     * @param variable sensitivity variable
-     * @return the sensitivity value associated with given function and given variable in state N.
+     * @param contingencyId the ID of the considered contingency
+     * @return the sensitivity value associated with given contingency id, use null for pre-contingency sensitivity values.
```

Comment:
Syntax: `with the given contingency ID`


To specify a user has to put null as a parameter to have pre-contingency sensitivity values, put it next to `@param`, not `@return`